ONBORDING.rst

```diff
@@ -0,0 +1,22 @@
+========================================
+Onbording to LSST Strong lensing project
+========================================
+
+This GitHub project originates from an inter-collaboration agreement between
+the two LSST Science collaborations:
+
+- `Strong Lensing Science Collaboration (SLSC) <https://sites.google.com/view/lsst-stronglensing?pli=1>`_
+- `Dark Energy Science Collaboration (DESC) <https://lsstdesc.org>`_
+
+
+link to pipeline overview (read only):
+https://docs.google.com/document/d/1dlwi7vP0jIL_miOF8VRiWd_0sALR4tklZlIIPuBJz9U/edit?usp=sharing
+
+link to template for sofware project
+link to ICA agreement (upon request)
+
+
+How to contribute:
+
+You can also follow the designated Slack channel on lsstc.slack.com  #strong-lensing
```

Comment:
Add 
For pipeline specific discussion, please join #sl-pipeline and #lensed-sn-integration